aspnetcore/signalr/background-service/sample/Server/ClockHub.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using HubServiceInterfaces;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Server
+{
+#region ClockHub
+    public class ClockHub : Hub<IClock>
+    {
+        public void SendTimeToClients(DateTime dateTime)
+        {
+            Clients.All.ShowTime(dateTime);
```

Comment:
This should be a Task returning method. Make it async and await this call.

Reply:
Fixed. Realized the Program.cs file had also been obliterated on some commit, so adding it back with these changes.